client/landing/browsehappy/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import { addQueryArgs } from 'calypso/lib/url';
+import Circle from './circle';
+import illustrationURL from './illustration.svg';
+import Logo from './logo-wide';
+import 'calypso/assets/stylesheets/style.scss';
+import './style.scss';
+
+const SUPPORTED_BROWSERS_LINK = 'https://wordpress.com/support/browser-issues/#supported-browsers';
+
+export default function Browsehappy( { from } ) {
+	const continueUrl = addQueryArgs( { bypassTargetRedirection: true }, from ?? '/' );
+
+	return (
+		<body className="browsehappy__body">
+			<nav className="browsehappy__nav">
+				<a href="https://wordpress.com" title={ __( 'WordPress.com' ) } rel="home">
+					<Logo />
+				</a>
+			</nav>
+			<main className="browsehappy__main" role="main">
+				<img src={ illustrationURL } alt="" />
+				<h1>{ __( 'Unsupported Browser' ) }</h1>
+				<p>{ __( 'Unfortunately this page may not work correctly in your browser.' ) }</p>
+				{ /* TODO: This mimics @wordpress/components button. Currently, wp components
+				does not compile in the server build because it is missing a peer dependency.
+				That problem will be resolved once the latest version of @wordpress/components
+				available in Calypso. We should switch to the Button component at that time. */ }
```

Comment:
Using `@wordpress/components` for SSR will be unfortunately harder than this. When I tried to use the latest `@wordpress/components` in a toy Next.js app, I got this error:

```
ReferenceError: CSS is not defined
    at space (/Users/jsnajdr/src/nextjs-blog/node_modules/@wordpress/components/build/ui/utils/space.js:38:3)
    at Object.<anonymous> (/Users/jsnajdr/src/nextjs-blog/node_modules/@wordpress/components/build/utils/config-values.js:72:42)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/Users/jsnajdr/src/nextjs-blog/node_modules/@wordpress/components/build/utils/style-mixins.js:61:44)
    at Module._compile (internal/modules/cjs/loader.js:1085:14) {
```
There is a `space()` helper function that calls `CSS.supports()`, which is available only in a browser environment. FYI @sarayourfriend

Reply:
That's too bad!